style.css

```
@@ -0,0 +1,326 @@
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 538px 200px 54px;
+  font: 12px/18px "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+}
+
+nav {
+  display: flex;
+  position: absolute;
+  width: 980px;
+  justify-content: space-between;
+  left: 184.5px;
+  top: 84px;
+  height: 36px;
+  border-radius: 4px;
+  background: #707070;
+  background: -webkit-gradient(linear, 0 0, 0 100%, from(rgba(0, 0, 0, 0.2)), color-stop(0.05, rgba(0, 0, 0, 0)), color-stop(0.97, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.45))), -webkit-gradient(linear, 0 0, 100% 0, from(rgba(0, 0, 0, 0.2)), color-stop(0.002, rgba(0, 0, 0, 0)), color-stop(0.998, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.2))), -webkit-gradient(linear, 0 100%, 0 0, from(#666), color-stop(0.5, #5e5e5e), color-stop(0.51, #707070), to(#808080));
+}
+
+nav ul {
+  flex: 8;
+}
+
+a {
+  text-decoration: none;
+}
+
+nav a {
+  display: block;
+  width: 36px;
+  visibility: hidden;
+}
+
+nav form {
+  flex: 1.2;
+  position: relative;
+}
+
+form input {
+  padding: 0;
+  position: absolute;
+  top: 9px;
+  width: 80%;
+  height: 19px;
+  left: 10px;
+  border-radius: 8px;
+  background: transparent;
+  border: 1px solid #525252;
+}
+
+form i {
+  z-index: 10;
+  color: black;
+}
+
+li {
+  display: inline-block;
+}
+
+nav li {
+  width: 12%;
+  height: 100%;
+  border-width: 0 1px 0 0;
+  border-style: solid;
+  border-color: #5b5b5b;
+}
+
+#nav-logo {
+  background: url(images/globalnav_text.png) no-repeat 50% -1.5%;
+  overflow: auto;
+}
+
+#nav-store {
+  background: url(images/globalnav_text.png) no-repeat 50% 14%;
+  overflow: auto;
+}
+
+#nav-mac {
+  background: url(images/globalnav_text.png) no-repeat 50% 28%;
+  overflow: auto;
+}
+
+#nav-ipod {
+  background: url(images/globalnav_text.png) no-repeat 50% 42.5%;
+  overflow: auto;
+}
+
+#nav-iphone {
+  background: url(images/globalnav_text.png) no-repeat 50% 57.2%;
+  overflow: auto;
+}
+
+#nav-ipad {
+  background: url(images/globalnav_text.png) no-repeat 50% 72%;
+  overflow: auto;
+}
+
+#nav-itunes {
+  background: url(images/globalnav_text.png) no-repeat 50% 86%;
+  overflow: auto;
+}
+
+#nav-support {
+  background: url(images/globalnav_text.png) no-repeat 50% 101.2%;
+  overflow: auto;
+}
+
+.hero-container {
+  background: url(/images/your_verse_hero.jpg) no-repeat 50% 100%;
+  margin-bottom: 1px;
+}
+
+.hero {
+  width: 980px;
+  margin: 120px auto;
+}
+
+.hero * {
+  padding: 3px 0;
+}
+
+.hero h1 {
+  position: relative;
+  top: 25px;
+}
+
+.hero .subheader2 {
+  height: 34px;
+  overflow: hidden;
+}
+
+.img {
+  display: grid;
+  grid-template: 1fr/ 1fr 1fr 1fr 1fr;
+}
+
+.img1 {
+  background: url(/images/promo_iphone5s.jpg);
+  margin: 0 1px;
+}
+
+.img2 {
+  background: url(/images/promo_iphone5c.jpg);
+  margin: 0 1px;
+}
+
+.img3 {
+  background: url(/images/promo_30_years.jpg);
+  margin: 0 1px;
+}
+
+.img4 {
+  background: url(/images/promo_supplier_responsibility.jpg);
```

Let's remove the first backslash `/`.

```suggestion
  background: url(images/promo_supplier_responsibility.jpg);
```